recipes/go-recipes/queue.go

```diff
@@ -82,29 +82,29 @@ func (q *Queue) Enqueue(trtr fdb.Transactor, item interface{}) (interface{}, err
 
 		return nil, nil
 	})
-	return i, e
+	return i, err
 }
 
 func (q *Queue) LastIndex(trtr fdb.Transactor) (interface{}, error) {
-	i, e := trtr.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
+	i, err := trtr.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
 		r, err := rtr.Snapshot().GetRange(q.QueueSS, fdb.RangeOptions{1, 0, true}).GetSliceWithError()
 		if len(r) == 0 {
 			return q.QueueSS.Pack(tuple.Tuple{0}), nil
 		}
 		return r[0].Key, err
 	})
-	return i, e
+	return i, err
 }
 
 func (q *Queue) FirstItem(trtr fdb.Transactor) (interface{}, error) {
-	i, e := trtr.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
+	i, err := trtr.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
```

Comment:
This could just be a `return`

Reply:
this is returning an error, so should i change the variable name

Reply:
i have renamed it from err to e

Reply:
It was fine as `err`, my suggestion was to directly return:
```go
func (q *Queue) FirstItem(trtr fdb.Transactor) (interface{}, error) {
	return trtr.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
```

But doesn't have to be done in this PR.